feature/manager/src/main/java/com/kinandcarta/create/proxytoggle/manager/view/composable/ProxyManagerScreen.kt

```diff
@@ -0,0 +1,315 @@
+@file:Suppress("TooManyFunctions")
+
+package com.kinandcarta.create.proxytoggle.manager.view.composable
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.clearAndSetSemantics
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.kinandcarta.create.proxytoggle.core.model.Proxy
+import com.kinandcarta.create.proxytoggle.core.theme.BlueyGrey
+import com.kinandcarta.create.proxytoggle.core.theme.ProxyToggleTheme
+import com.kinandcarta.create.proxytoggle.core.theme.StatusLabelTextStyle
+import com.kinandcarta.create.proxytoggle.manager.R
+import com.kinandcarta.create.proxytoggle.manager.viewmodel.ProxyManagerViewModel
+import com.kinandcarta.create.proxytoggle.manager.viewmodel.TextFieldState
+
+@Composable
+fun ProxyManagerScreen(
+    viewModel: ProxyManagerViewModel = viewModel(),
+    useVerticalLayout: Boolean
+) {
+    val uiState by viewModel.uiState
+
+    ProxyManagerScreenContent(
+        useVerticalLayout = useVerticalLayout,
+        darkTheme = uiState.darkTheme,
+        onToggleTheme = viewModel::toggleTheme,
+        proxyEnabled = uiState.proxyEnabled,
+        onToggleProxy = viewModel::toggleProxy,
+        addressState = uiState.addressState,
+        onAddressChanged = viewModel::onAddressChanged,
+        portState = uiState.portState,
+        onPortChanged = viewModel::onPortChanged,
+        onForceFocusExecuted = viewModel::onForceFocusExecuted
+    )
+}
+
+@Suppress("LongParameterList")
+@Composable
+fun ProxyManagerScreenContent(
```

Comment:
If the list of parameters is getting out of hand consider creating a single lambda to pass back user events modelled using a sealed class.

Reply:
done!